vmdb/lib/tasks/test_metrics.rake

```diff
@@ -4,7 +4,7 @@
 #   original here: https://github.com/ManageIQ/metric_fu/blob/master/tasks/metric_fu.rake
 require_relative "./evm_test_helper"
 
-if defined?(RSpec)
+if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
```

Comment:
Maybe we should have a loader to make this more obvious:

lib/tasks/rspec_task_loader.rake
```ruby
if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
  Dir.glob(Rails.root.join("lib", "tasks", "rspec", "*.test_rake")).each { |r| load f }
end
```

And move all of the rspec dependent rake tasks into lib/tasks/rspec/*.test_rake or better named place?

thoughts? @Fryguy @kbrock